x/interchainquery/keeper/msg_submit_query_response_test.go

```diff
@@ -0,0 +1,165 @@
+package keeper_test
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/bech32"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	_ "github.com/stretchr/testify/suite"
+	"github.com/tendermint/tendermint/proto/tendermint/crypto"
+
+	"github.com/Stride-Labs/stride/x/interchainquery/types"
+)
+
+type MsgSubmitQueryResponseTestCase struct {
+	validMsg types.MsgSubmitQueryResponse
+	goCtx    context.Context
+	query    types.Query
+}
+
+func (s *KeeperTestSuite) SetupMsgSubmitQueryResponse() MsgSubmitQueryResponseTestCase {
+
+	// set up IBC
+	hostChainId := "GAIA"
```

Comment:
```suggestion
	const HostChainId := "GAIA"
```
nit: make constant outside of this function

Reply:
good idea, done